package.json

```diff
@@ -21,20 +21,24 @@
     "vue-router"
   ],
   "dependencies": {
+    "@storybook/addon-actions": "^3.2.10",
+    "global": "^4.3.2",
     "prop-types": "^15.5.10",
-    "react": "^0.14.7 || ^15.0.0",
     "react-router": "^3.0.0 || ^4.0.0",
     "vue": "^2.4.4",
-    "vue-router": "^2.7.0",
-    "@storybook/addon-actions": "^3.2.10"
+    "vue-router": "^2.7.0"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
     "eslint": "^3.19.0",
     "eslint-plugin-react": "^7.0.1",
+    "react": "^16.0.0",
     "react-scripts": "1.0.7"
   },
+  "peerDependencies": {
+    "react": "^0.14.7 || ^15.0.0 || ^16.0.0"
```

Comment:
With the exception of adding the new major version of React for me moving the React dependency from `dependencies` to `peerDependencies` does not make any difference. Even if I'd like to test modifications to this package using `npm link` I cannot see a direct relationship of those things and I still have the error:
```
Uncaught Error: Accessing nonexistent addons channel
```
Maybe @ndelangen has an explaination for that.

Reply:
adding it to peer dependencies ensures consumers do not run the app with multiple version of react. Its a recommended way of bulding third-party components. I've moved the local version of react for testing locally to devDependencies.